doc/source/notebooks/03_node_properties.ipynb

```diff
@@ -310,16 +317,22 @@
    "execution_count": 5,
    "metadata": {},
    "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/gpfs/bbp.cscs.ch/data/scratch/proj30/home/herttuai/NSETM_2185/venv/lib/python3.10/site-packages/pandas/plotting/_matplotlib/core.py:1256: UserWarning: No data for colormapping provided via 'c'. Parameters 'cmap' will be ignored\n",
```

Comment:
This warning is new, is it possible to fix it? (here or in a different PR)

Reply:
I added code to temporarily ignore it since I didn't find an immediate solution.